source/lac/trilinos_sparse_matrix.cc

```diff
@@ -180,6 +180,7 @@ namespace TrilinosWrappers
           colnum_cache->data()));
       value_cache->resize(ncols);
       colnum_cache->resize(ncols);
```

Comment:
If we know that `ncols==colnums` then we don't need these two lines anymore.

Reply:
You are right :+1: